test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -104,69 +150,97 @@ public void testConcurrentScanConsistency() throws Exception {
-        assertTrue(stats.verified > 0);
-        // These scans were running concurrently with writes, so a scan will see more data than what
-        // was written before the scan started.
-        assertTrue(stats.scanned > stats.verified);
-      }
+    for (Future<ScanStats> scanTask : scanTasks) {
+      var stats = scanTask.get();
+      log.debug(String.format("Scanned:%,d verified:%,d", stats.scanned, stats.verified));
+      checkTrue(stats.verified > 0);
+      // These scans were running concurrently with writes, so a scan will see more data than what
+      // was written before the scan started.
+      checkTrue(stats.scanned > stats.verified);
+    }
+
+    log.debug(tableOpsTask.get());
+
+    var stats1 = scanData(testContext, random, new Range(), false);
+    var stats2 = scanData(testContext, random, new Range(), true);
+    var stats3 = batchScanData(testContext, new Range());
+    log.debug(
+        String.format("Final scan, scanned:%,d verified:%,d", stats1.scanned, stats1.verified));
+    checkTrue(stats1.verified > 0);
+    // Should see all expected data now that there are no concurrent writes happening
+    checkEquals(stats1.scanned, stats1.verified);
+    checkEquals(stats2.scanned, stats1.scanned);
+    checkEquals(stats2.verified, stats1.verified);
+    checkEquals(stats3.scanned, stats1.scanned);
+    checkEquals(stats3.verified, stats1.verified);
+
+    executor.shutdownNow();
+  }
 
-      log.debug(tableOpsTask.get());
-
-      var stats1 = scanData(testContext, random, new Range(), false);
-      var stats2 = scanData(testContext, random, new Range(), true);
-      var stats3 = batchScanData(testContext, new Range());
-      log.debug(
-          String.format("Final scan, scanned:%,d verified:%,d", stats1.scanned, stats1.verified));
-      assertTrue(stats1.verified > 0);
-      // Should see all expected data now that there are no concurrent writes happening
-      assertEquals(stats1.scanned, stats1.verified);
-      assertEquals(stats2.scanned, stats1.scanned);
-      assertEquals(stats2.verified, stats1.verified);
-      assertEquals(stats3.scanned, stats1.scanned);
-      assertEquals(stats3.verified, stats1.verified);
-    } finally {
-      executor.shutdownNow();
+  /**
+   * Checks if b is true. Uses JUnit assert if inTestingContext, otherwise checks if b is true and
+   * throws an exception if not.
+   *
+   * @param b The boolean checked
+   * @throws Exception
+   */
+  private static void checkTrue(boolean b) throws Exception {
+    if (inTestingContext) {
+      assertTrue(b);
+    } else if (!b) {
+      throw new Exception("Failed assertion");
+    }
+  }
+
+  /**
+   * Checks if l1 and l2 are equal. Uses JUnit assert if inTestingContext, otherwise checks if l1
+   * and l2 are equal and throws an exception if not.
+   *
+   * @param l1 One of the two longs to be compared
+   * @param l2 One of the two longs to be compared
+   * @throws Exception
```

Comment:
I think these `@throws` tags without a description are causing the build to fail. I think they can be removed 
```suggestion
   * @throws Exception
```